serverless.yml

```diff
@@ -2,6 +2,7 @@ service: lbh-social-care
 
 provider:
   name: aws
+  variablesResolutionMode: 20210326
```

Comment:
I'm pretty sure this should be at the root...

https://www.serverless.com/framework/docs/providers/aws/guide/variables#reference-variables-using-the-ssm-parameter-store

https://github.com/LBHackney-IT/lbh-core-pathway-pilot/blob/main/serverless.yml#L3

Reply:
My bad, fixed!